check_postgres.pl

```diff
@@ -4413,8 +4418,16 @@ sub check_custom_query {
             }
             $goodrow++;
             if ($perfname) {
-                $db->{perf} .= sprintf ' %s=%s;%s;%s',
-                    perfname($perfname), $r->{$perfname}, $warning, $critical;
+                if ($grandtotal > 1) {
+                    $perfdata = sprintf ' %s=%s;%s;%s',
+                        perfname($r->{$perfname}), $result, $warning, $critical;
+                    if ($perfdata ne $db->{perf}){
+                        $db->{perf} .= $perfdata;
```

Comment:
If more than one row is returned, store the perfdata to a temporary variable and check if it has already been appended to the output.